moveit_configs_utils/moveit_configs_utils/moveit_configs_builder.py

```diff
@@ -51,17 +51,63 @@
 """
 
 from pathlib import Path
-from typing import Optional, List
+from typing import Optional, List, Union
 import logging
+import xacro
+
 import re
 from ament_index_python.packages import get_package_share_directory
-
 from launch_param_builder import ParameterBuilder, load_yaml, load_xacro
-
+from jinja2 import Template
+from urdf_parser_py.urdf import URDF
+from xml.dom.minidom import Document, parseString
 
 moveit_configs_utils_path = Path(get_package_share_directory("moveit_configs_utils"))
 
 
+def generate_fake_system_description(
```

Comment:
Suggest renaming this

```suggestion
def generate_fake_controllers(
```